tests/workspace/page_view.rs

```diff
@@ -39,3 +42,53 @@ async fn get_page_view() {
     .unwrap();
   assert_eq!(resp.data.row_data.len(), 0);
 }
+
+#[tokio::test]
+async fn create_new_document_page() {
+  let (c, _user) = generate_unique_registered_user_client().await;
+  let workspaces = c.get_workspaces().await.unwrap();
+  assert_eq!(workspaces.len(), 1);
+  let workspace_id = workspaces[0].workspace_id;
+  let folder_view = c
+    .get_workspace_folder(&workspace_id.to_string(), Some(2), None)
+    .await
+    .unwrap();
+  let general_space = &folder_view
+    .children
+    .into_iter()
+    .find(|v| v.name == "General")
+    .unwrap();
+  let page = c
+    .create_workspace_page_view(
+      workspace_id,
+      &CreatePageParams {
+        parent_view_id: general_space.view_id.clone(),
+        layout: ViewLayout::Document,
+      },
+    )
+    .await
+    .unwrap();
```

Comment:
I have experienced flaky test in template duplication test cases, which use the same mechanism to broadcast all updates to client. I solved at by waiting for 1 second before the next api call. If you experienced flaky test scenario in future test runs, can consider to use this technique.